code/src/components/CheeseInfo.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+
+export const CheeseInfo = () => {
+  const scanned = useSelector((state) => state.products.product)
+
+  if (scanned.product.nutriscore_data.is_cheese) {
```

Comment:
I think you need to check whether the product has `nutriscore_data` here too. I scanned some butter which didn't have any nutritional data added, and it caused an exception here.

```js
if (scanned.product.nutriscore_data && scanned.product.nutriscore_data.is_cheese) {

}
```